docs/source/gke.md

```diff
@@ -129,17 +127,18 @@ kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-ad
 
 If you don't have Helm installed, Go to the [helm docs](https://docs.helm.sh/using_helm/#installing-helm) to get the binary for your distro.
 
-#### Install RAID DaemonSet
+#### Install xfs-format DaemonSet
 
-To combine the local disks together in RAID0 arrays, we deploy a DaemonSet to do the work for us.
+To run Scylla, it is necessary to convert ephemeral storage's filesystem to xfs. Deploy the xfs-format DaemonSet to have it taken care of.
+Unfortunately, GKE is only able to provision the ephemeral storage with `ext4` filesystem while Scylla requires `xfs` filesystem. Deploying the `xfs-format` DaemonSet will format the storage as `xfs` and prevent GKE from reformatting it back to `ext4.
```

Comment:
We should make it clear that this is super hacky and, although we did our best to limit the risk, the is still some in case google changes their logic.